app/models/bishop.rb

```diff
@@ -1,2 +1,7 @@
 class Bishop < Piece
+
+  def valid_move?(x, y) #NO CAPTURE/COLLISION LOGIC YET
```

Comment:
Got it.  Was generally aware that in-line commenting like that was a no-go in production but I wasn't sure the best way to inform the other team members of what was going on -- will take a look at Doc Block and explore converting to that.